templates/trackingCode.twig

```diff
@@ -1,4 +1,4 @@
-<div piwik-content-block anchor="tagmanager" content-title="{{ 'TagManager_TagManager'|translate }}">
+<div piwik-content-block anchor="tagmanager" content-title="{{ 'TagManager_MatomoTagManager'|translate }}">
```

Comment:
Does this new translation need to be added to `TagManager.php` since it's being used in this view template?

Reply:
@snake14 Not needed for twig